Source/Mosa.Tool.Explorer/MainForm.cs

```diff
@@ -467,6 +467,11 @@ private void CompileAll()
 		if (Compiler == null)
 			return;
 
+		if (MosaSettings.EmitBinary & !Directory.Exists(MosaSettings.DefaultFolder))
```

Comment:
Seems like you forgot an ampersand here.

Reply:
Nice catch!